lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -1,37 +1,41 @@
 // lms - views - user/student dashboard
 // ====================
 
+// Table of Contents 
+// * +Dashboard - Sidebar
+// * +Dashboard - Course Listing
+// * +Dashboard - Course Item
+// * +Misc - Uncategorized
+// * +Dashboard - Banner
+
+
+// +Dashboard - Sidebar
+// ==================== 
 .dashboard {
   @include clearfix();
   padding: ($baseline*2) 0 0 0;
 
   .profile-sidebar {
-    background: transparent;
-    @include float(left);
-    @include margin-right(flex-gutter());
+    @include float(right);
+    @include margin-left(flex-gutter());
     width: flex-grid(3);
     background: transparent;
-    box-shadow: 0 0 1px $shadow-l1;
 
-    header.profile {
+    .profile {
       @include box-sizing(border-box);
       border: 1px solid $border-color-2;
       border-radius: ($baseline/4) ($baseline/4) 0 0;
       width: flex-grid(12);
-      background: $dashboard-profile-color;
 
       .user-name {
+        @extend %t-title6;
```

Comment:
this is a little too small for the space - i would bump it up to %t-title5 or reduce the padding.